Components/Slider/slider.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Slider</title>
+    <link rel="stylesheet" href="../main.css" />
+    <link rel="stylesheet" href="slider.css" />
+  </head>
+  <body>
+    <div class="flex-container">
+      <section class="range-slider">
+        <input value="500" min="500" max="50000" step="500" type="range" />
```

Comment:
You can make the slider functional, meaning show the current value of the slider on the view, along with its min and max values.